### PR TITLE
upgrade axios to 0.21.1

### PR DESCRIPTION
**Pull request by FeelyChau**
Upgrade axios for security.

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@pipcook/pipcook-core": "^1.3.0",
     "@pipcook/sdk": "^1.3.0",
-    "axios": "^0.18.1",
+    "axios": ">=0.21.1",
```
Comment (yorkie) [inline packages/cli/package.json:28]:
```suggestion
    "axios": "^0.21.1",
```

**Comment by WenheLI** [comment]
Should use `^` for all versions

**Comment by FeelyChau** [comment]
Fixed, @WenheLI @yorkie may I ask you to review again?

**Review by WenheLI — APPROVED** [review]
LGTM

**Pull request merged**